deployment/kubernetes/livenessprobe-sidecar.yaml

```diff
@@ -50,4 +50,5 @@
     args:
     - --v=4
     - --csi-address=/csi/csi.sock
+    - --logging-format=json
```

Comment:
Should the default be json or still text?. A concern about this line is that when a CSI Driver author looks at this line they might change the flag value and that might break things like log ingestion downstream if for example it's configured to read doing regex on text instead of json

Reply:
It seems to me that changing the default output format to json might cause problems in the user's system, so perhaps the default output format should remain as text.
(Since the content of the message has also been changed for structured logging, I think an announcement about that is necessary.)